.gitignore

```diff
@@ -33,8 +33,6 @@ docker-compose.yml
 /.env
 extras/
 release/
-build/
-cli/release/
```

Comment:
until we did not unify I think we should let it in gitignore ...

this is for another pull :)

Reply:
You're right, I changed only `build/` to `/build/` as the gitignore was wrong at that point and ignored the build folder inside cli as well.